joern-cli/src/main/scala/io/joern/joerncli/CpgBasedTool.scala

```diff
@@ -52,4 +52,16 @@ object CpgBasedTool {
     }
   }
 
+  def exitIfInvalid(outDir: String, cpgFileName: String): Unit = {
+    if (File(outDir).exists)
+      exitWithError(s"Output directory `$outDir` already exists.")
```

Comment:
in most other situations we delete the existing output...

Reply:
(personally I prefer if tools just override so I can invoke them multiple times without having to worry about deleting files etc... - ymmv)